src/__test__/actions/profile/ProfileActions.test.js

```diff
@@ -0,0 +1,227 @@
+import moxios from 'moxios';
+import axios from 'axios';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { fetchProfile, editUserProfile } from '../../../actions/profileActions';
+
+
+const middleWare = [thunk];
+const mockStore = configureStore(middleWare);
+
+describe('fetch profile', () => {
+  beforeEach(() => {
+    moxios.install(axios);
+  });
+  afterEach(() => {
+    moxios.uninstall(axios);
+  });
+
+  it('should be able to fetch profile', () => {
+    const props = {
+      history: {
+        push: jest.fn(),
+      },
+    };
+    const dataReturned = {
+      profile: {
+        first_name: 'first',
+        last_name: 'last',
+        image: 'https://firebasestorage.googleapis.com/v0/b/ah-frontend-invictus.appspot.com/o/images%2FPhotoGrid_1549744503436%20(1).jpg?alt=media&token=15234b51-ca88-488e-8bc3-494d6e717ac8',
+        bio: 'am so cool',
+        following: false,
+      },
+    };
+
+    const Process = [
+      {
+        payload: dataReturned,
+        type: 'LOAD_PROFILE',
+      },
+    ];
+
+    moxios.wait(() => {
+      const request = moxios.requests.mostRecent();
+      request.respondWith(
+        {
+          response: dataReturned,
+        },
+      );
+    });
+    const store = mockStore({});
+    return store.dispatch(fetchProfile(props))
+      .then(() => {
+        expect(store.getActions()).toEqual(Process);
+      });
+  });
+
+
+  //   it('should be able to return an error on failure to fetch profile', () => {
```

Comment:
Hi @NicholusMuwonge, kindly remove this commented out code.

Reply:
I have reacted to this please